frontend/templates/views/examples/documentation.j2

```diff
@@ -1,5 +1,7 @@
 {% extends '/layouts/default.j2' %}
 {% set inline_preview = not doc.example.document.metadata.hidePreview %}
+{% do doc.amp_dependencies.add('amp-iframe', '0.1') %}
```

Comment:
`amp-iframe` on the example documentation pages is only needed for story preview.